### ISSUE TITLE
atmega8 error on programing

### ISSUE THREAD

**Issue by sarath-crypto** [crawford/bootjack#1]
avrdude: ERROR: address 0x1e010 out of range at line 2 of bootjack.hex
Minor change in done is
static inline bool app_present(void)
{
        //const __flash volatile uint16_t *start = CONFIG_APP_START_ADDR;
        const volatile uint16_t *start = CONFIG_APP_START_ADDR;
        return (*start != 0xFFFF);
}

I am using atmega 8. Any guidance will be helpful to me.

$ avr-gcc --version
avr-gcc (GCC) 5.4.0
Copyright (C) 2015 Free Software Foundation, Inc.

**Comment by crawford** [comment]
Sorry for the delay. It's been a while since I've looked at this project!

That "magic" address comes from: https://github.com/crawford/bootjack/blob/65a976d0904f368c19fa726d272a4bbd8bc32af5/config/default.mk#L5

I cannot remember which specific processor I was using when I developed the project, but you'll need to update that value to match the start of the boot flash section for your processor. Based on that error message (and the fact that it's an ATmega 8), I'm guessing your flash is smaller.
